api_tests/registrations/views/test_registration_detail.py

```diff
@@ -66,3 +69,44 @@ def test_do_not_return_node_detail(self):
         res = self.app.get(url, auth=self.user.auth, expect_errors=True)
         assert_equal(res.status_code, 400)
         assert_equal(res.json['errors'][0]['detail'], "This is not a registration.")
+
+    def test_retractions_display_limited_fields(self):
+        registration = RegistrationFactory(creator=self.user, project=self.public_project, public=True)
+        url = '/{}registrations/{}/'.format(API_BASE, registration._id)
+        retraction = RetractedRegistrationFactory(registration=registration, user=registration.creator)
+        retraction.justification = 'We made a major error.'
+        retraction.save()
+        res = self.app.get(url, auth=self.user.auth, expect_errors=True)
+        assert_equal(res.status_code, 200)
+
+        assert_items_equal(res.json['data']['attributes'], {
```

Comment:
Hm, the docs make it sound like this assert_items_equal was meant to be used to compare sequences (e.g. lists and tuples) which may have different item ordering. I'm not sure what the behavior is for dicts. If you change one of the *values* in the dict, does it fail as expected?